app/src/app/api/v0/auth/verify/route.ts

```diff
@@ -25,14 +25,14 @@ export const GET = apiHandler(
       throw createHttpError.NotFound("User not found!");
     }
 
-    if (!process.env.RESET_TOKEN_SECRET) {
+    if (!process.env.VEFIFICATION_TOKEN_SECRET) {
```

Comment:
Typo in all of these
```suggestion
    if (!process.env.VERIFICATION_TOKEN_SECRET) {
```